package.json

```diff
@@ -88,13 +87,13 @@
     "kleur": "3.0.3",
     "lint-staged": "8.2.1",
     "nock": "12.0.3",
-    "npm-run-all": "^4.1.5",
+    "npm-run-all": "4.1.5",
     "prettier": "2.0.5",
     "rimraf": "3.0.2",
     "selfsigned": "1.10.7",
     "standard-version": "8.0.0",
     "supertest": "4.0.2",
-    "typescript": "4.0.2",
+    "typescript": ">=3.3.1 <3.10.0",
```

Comment:
for compatibility with @typescript-eslint

Reply:
I'd prefer stay on `v4`, it is supported already https://github.com/typescript-eslint/typescript-eslint/issues/2260#issuecomment-677853883

Reply:
Let's update to the latest, was released yesterday

Reply:
eslint didnt run with tsv4

Reply:
I tried with upgrading all eslint-related dependencies and typescript. That worked but then I have to fix a bunch of eslint errors.

Reply:
No problem, let's keep it on v3. We can do it later.

Reply:
Thanks